ecommerce/frontend/src/components/product_card/ProductCard.js

```diff
@@ -1,26 +1,59 @@
 import React from 'react'
 import { Button, Rate } from 'antd'
-import { PlusCircleOutlined, HeartOutlined } from '@ant-design/icons'
+import { PlusCircleOutlined, HeartOutlined, EditOutlined, DeleteOutlined } from '@ant-design/icons'
 import { Link } from 'react-router-dom'
 import './ProductCard.less'
 import { useAppContext } from '../../context/AppContext'
 import { round, truncate } from '../../utils'
+import { ISO_8601 } from 'moment'
```

Comment:
Not sure what this is necessary for? I never had to use it personally.

Reply:
It is left from the time you first mentioned about the Date format that Backend sends. I had imported it then, but it is unnecessary now.